SAGA/objects/calc_sfr.py

```diff
@@ -6,7 +6,7 @@
 __all__ = ["calc_SFR_NUV", "calc_SFR_Halpha"]
 
 
-def calc_SFR_NUV(NUV_mag, NUV_mag_err, dist_mpc, internal_ext=1.0):
+def calc_SFR_NUV(NUV_mag, NUV_mag_err, dist_mpc, internal_ext=0.83):
```

Comment:
Update to 0.9.   This value comes from converting Halpha Balmer Decrement value (BD = 3.25) into A_NUV assuming a Calzetti00 reddening curve and R_V = 3.76 from Battisti 2016.